frontend/components/Filter/OrganiserFilterDialog.tsx

```
@@ -380,25 +337,53 @@ export default function OrganiserFilterDialog({
+              {dateRange.from.toLocaleDateString()}
+              {dateRange.to && ` - ${dateRange.to.toLocaleDateString()}`}
+            </p>
+          )}
         </div>
-        <div className="mt-5 w-full flex items-center">
+        <div className="mt-4 w-full flex items-center">
           <button
-            className="hover:underline cursor-pointer rounded-md border border-transparent bg-blue-100 px-4 py-2 text-sm font-medium text-blue-900 hover:bg-blue-200 focus:outline-none focus-visible:ring-2 focus-visible:ring-blue-500 focus-visible:ring-offset-2"
+            className="hover:underline cursor-pointer rounded-md bg-core-hover text-core-text px-4 py-2 text-sm font-medium border-[1px] border-core-text transition-all duration-300 transform"
             onClick={handleClearAll}
           >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)